Gruntfile.js

```diff
@@ -81,6 +82,16 @@ module.exports = function (grunt) {
     config: config,
     loadGruntTasks: {
       pattern: ['grunt-*', '@*/grunt-*', 'gruntify-*', '@*/gruntify-*', 'intern']
+    },
+    preMerge(config, data) {
```

Comment:
So did we decide to leave files in this directory camelCase if need be? I'm fine with that, its the least bad of all the alternatives.